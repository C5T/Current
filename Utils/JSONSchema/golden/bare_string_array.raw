{"Array":{"element":{"String":{"values":{"bar":1,"baz":1,"foo":1},"instances":3,"nulls":0}},"instances":1,"nulls":0}}