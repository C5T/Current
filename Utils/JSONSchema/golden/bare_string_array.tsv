Field	Type	Set	Unset/Null	Values	Details
Schema	Array	1	0
Schema[]	String	3	0	3 distinct values	`bar` : 1, `baz` : 1, `foo` : 1
